test/allowlist_for_publicAPI.json

```diff
@@ -1026,8 +1026,17 @@
     "randn"
   ],
   "torch.types": [
-    "Any",
```

Comment:
New entries are not allowed in this file. Please clean up the public API definition in that file based on the suggestion from the error message.
In this case, you need to fix the `__all__` list which doesn't contain the right things!